src/CppParser/Parser.cpp

```diff
@@ -903,6 +903,14 @@ void Parser::WalkRecord(const clang::RecordDecl* Record, Class* RC)
         ReadClassLayout(RC, Record, CharUnits(), true);
     }
 
+    if (Record->redecls_begin() != Record->redecls_end()) {
+        for (auto redecl : Record->redecls()) {
+            auto dec = WalkDeclaration(redecl);
+            if (dec != nullptr)
```

Comment:
It is needed for safety. I encountered that it could happen that the iterator for Redeclarations iterates over a list with empty entries.